pkg/gadgets/trace/tcpdrop/tracer/bpf/tcpdrop.bpf.c

```diff
@@ -0,0 +1,134 @@
+// SPDX-License-Identifier: GPL-2.0
+//
+// Based on tcpdrop(8) from BCC
+//
+// Copyright 2018 Netflix, Inc.
+// 30-May-2018    Brendan Gregg   Created this.
+// 15-Jun-2022    Rong Tao        Add tracepoint:skb:kfree_skb
+// Copyright 2023 Microsoft Corporation
+
+#include <vmlinux/vmlinux.h>
+
+#include <bpf/bpf_helpers.h>
+#include <bpf/bpf_core_read.h>
+#include <bpf/bpf_tracing.h>
+#include <bpf/bpf_endian.h>
+
+#include "tcpdrop.h"
+
+/* Define here, because there are conflicts with include files */
```

Comment:
I took this sentence from other gadgets. It was first introduced by commit 5c9e0ea56ce4a8257d07a010b6deaef51321b990.

I think it comes from gadgets that include vmlinux.h: all structs of the kernel are imported in vmlinux.h but not defines like AF_INET. But if we include the header file for AF_INET, it will include other structs that are also defined in vmlinux, hence the conflict.